VKApp/VKApp/Service/Operation/ParseDataOperation.swift

```diff
@@ -0,0 +1,21 @@
+// ParseDataOperation.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Парсинг данных
+final class ParseDataOperation: Operation {
+    // MARK: Public Property
+
+    var outputData: Group?
```

Comment:
```suggestion
    var outputData: Group?
```
у всех тут массив обычно